lawnchair/src/app/lawnchair/ui/preferences/destinations/QuickstepPreferences.kt

```diff
@@ -94,3 +103,16 @@ fun QuickstepPreferences() {
         }
     }
 }
+
+@Composable
+fun SettingsIsIgnoreWarning() {
+    Surface(
+        modifier = Modifier.padding(horizontal = 16.dp),
+        shape = MaterialTheme.shapes.large,
+        color = androidx.compose.material3.MaterialTheme.colorScheme.errorContainer,
+    ) {
+        WarningPreference(
+            text = "You are currently using a development build, this settings will be ignored by Lawnchair"
```

Comment:
Fix code styling: Spotless

```suggestion
            text = "You are currently using a development build, this settings will be ignored by Lawnchair",
```